infrastructure/docker/git-ssh/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM alpine:edge
```

Comment:
I would use a "3.18" here just like in the other docker files

Reply:
done